README.md

```diff
@@ -177,6 +177,7 @@ See also *[HackingThe.cloud](https://hackingthe.cloud/)*.
 ## Collaboration Tools
 
 * [Dradis](https://dradisframework.com) - Open-source reporting and collaboration tool for IT security professionals.
+* [Hexway Hive](hexway.io/hive/) - Self hosted pentest collaboration, multi-source data aggregation and reporting framework with customizable templatess and methodologies
```

Comment:
```suggestion
* [Hexway Hive](https://hexway.io/hive/) - Commercial collaboration, data aggregation, and reporting framework for red teams with a limited free self-hostable option.
```

At a minimum, this needs to be actual URI, not just a domain and path component.

The tool looks like many others but I'd list it if its description did not overpromise what's actually available; it's important to note non-free limitations given how much of the rest of the listings here are not similarly hobbled.